packages/cactus-plugin-ledger-connector-polkadot/src/main/json/openapi.json

```diff
@@ -0,0 +1,781 @@
+              "$ref": "#/components/schemas/web3SigningCredential",
+              "nullable": false
+            },
+            "metadata": {
+              "oneOf": [
+                {
+                  "type": "string"
+                }
+              ]
+            },
+            "contractAddress": {
+              "type": "string",
+              "nullable": false
+            },
+            "methodName": {
+              "description": "The name of the contract method to invoke.",
+              "type": "string",
+              "nullable": false,
+              "minLength": 1,
+              "maxLength": 2048
+            },
+            "gasLimit": {
+              "type": "object",
+              "required": ["refTime", "proofSize"],
+              "properties": {
+                "refTime": {
+                  "type": "integer",
+                  "minimum": 0
+                },
+                "proofSize": {
+                  "type": "integer",
+                  "minimum": 0
+                }
+              }
+            },
+            "storageDepositLimit": {
+              "oneOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "integer",
+                  "minimum": 0
+                }
+              ],
+              "nullable": true
+            },
+            "balance": {
+              "oneOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "integer",
+                  "minimum": 0
+                }
+              ]
+            },
+            "params": {
+              "description": "The list of arguments to pass in to the contract method being invoked",
+              "type": "array",
+              "default": [],
+              "items": {}
+            }
+          }
+        },
+        "InvokeContractResponse": {
+          "type": "object",
+          "required": ["success"],
+          "properties": {
+            "callOutput": {},
+            "success": {
+              "type": "boolean",
+              "nullable": false
+            },
+            "txHash": {
+              "type": "string",
+              "nullable": false
+            },
+            "blockHash": {
+              "type": "string",
+              "nullable": false
+            }
+          }
+        }
+      }
+    },
+    "paths": {
+    "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-polkadot/get-prometheus-exporter-metrics": {
+      "get": {
+        "x-hyperledger-cactus": {
+          "http": {
+            "verbLowerCase": "get",
+            "path": "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-polkadot/get-prometheus-exporter-metrics"
+          }
+        },
+        "operationId": "getPrometheusMetrics",
+        "summary": "Get the Prometheus Metrics",
+        "parameters": [],
+        "responses": {
+          "200": {
```

Comment:
Please add description of error responses as well (similar to this - https://github.com/hyperledger/cacti/blob/main/packages/cactus-plugin-ledger-connector-ethereum/src/main/json/openapi.json#L1150). This applies to other endpoints as well

Reply:
done